api/v2/guides/google-ss-integration.md

```diff
@@ -0,0 +1,81 @@
+
+With this framework, the following steps occur:
+
+1. Publishers push user ID signals (advertising tokens) to the secure signals feature.
+2. The secure signals feature caches them on the client side and then transparently passes them to Google Ad Manager.
+3. Google Ad Manager uses the UID2 tokens to make bid requests -- forwarding the tokens to approved bidders within Google Adx based on the publisher's preferences.
+
+## Allow Secure Signal Sharing
+
+For your Google Ad Manager account to be eligible to receive encrypted UID tokens, you must make sure that encrypted signals are properly shared with third-party bidders on your Google Ad Manager account.
+
+For details, see [Share encrypted signals with bidders](https://support.google.com/admanager/answer/10488752) (Google reference documentation) and then follow the steps in [Use a third-party signal provider](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/securesignals) to switch on UID2 as your signal provider.
+
+## Publisher Integration
+
+When an encrypted signal is cached, the secure signals feature does not execute the handler to generate a new signal. Because of this, it is necessary to clear the cache before login and after logout.
+
+Since the secure signals feature does not provide a way to delete or invalidate a specific ID, publishers must call the `window.googletag.secureSignalProviders.clearAllCache()` function to clear all shared encrypted signals as part of their login/logout workflows.
+
+The following is an example of calling the `window.googletag.secureSignalProviders.clearAllCache()` function:
+
+```
+window.googletag = window.googletag || { cmd: [] }; 
+window.googletag.cmd.push(function () { 
+   window.googletag.secureSignalProviders.clearAllCache(); 
+});
+```
+
+### Server-Only Integration
+
+So that it can share encrypted signals, the hosted auto-loaded secure signals script must be able to make an asynchronous call to the `window.getUid2AdvertisingToken` function and, in response, receive `advertising_token` as a string.
+
+It's important to make sure that the identity token is fresh. For a server-side integration, we recommend making a call to the {{/token/refresh}} endpoint](https://github.com/IABTechLab/uid2docs/blob/main/api/v2/endpoints/post-token-refresh.md#post-tokenrefresh) to get a fresh [identity token](https://github.com/IABTechLab/uid2docs/blob/main/api/v2/endpoints/post-token-refresh.md#decrypted-json-response-format).
+
+The following code is an example of how you could do this.
+
+```
+window.getUid2AdvertisingToken = async () => {
+  // Make a call to get a fresh identity token which could last for at least 12 hours.
+  const identity = await getFreshIdentity
```

Comment:
sorry, my bad , should be   const identity = await getFreshIdentity()

Reply:
Fixed. Thx! Update coming in a minute.